plugins/experimental/ja3_fingerprint/ja3_fingerprint.cc

```diff
@@ -270,7 +270,14 @@ client_hello_ja3_handler(TSCont contp, TSEvent event, void *edata)
 {
   TSVConn ssl_vc = reinterpret_cast<TSVConn>(edata);
   switch (event) {
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+  case TS_EVENT_SSL_SERVERNAME: {
+#elif OPENSSL_VERSION_NUMBER >= 0x10101000L
   case TS_EVENT_SSL_CLIENT_HELLO: {
+#else
+#error OpenSSL cannot be 1.1.0
+#endif
+  
```

Comment:
Probably don't need the ifdefs here.  Ifdefing in the set up code should suffice. You could just have both cases here and make a comment about they they are both here.